components/Table/Table.tsx

```diff
@@ -1,30 +1,34 @@
-import { FC } from "react"
+import { FC } from 'react'
 
 import {
   Table,
   Tbody,
   Th,
   Thead,
   Tr,
-  useBreakpointValue
+  useBreakpointValue,
 } from '@ironfish/ui-kit'
-import size from "byte-size"
+import size from 'byte-size'
 
-import DataRowSmall from "./DataRowSmall"
-import DataRowLarge from "./DataRowLarge"
-import RowItem from "./RowItem"
-import RowItemSpin from "./RowItemSpin"
-import { CommonTableProps } from "./types"
-import RowItemWrapperSmall from "./RowItemWrapperSmall"
-import RowItemWrapperLarge from "./RowItemWrapperLarge"
+import DataRowSmall from './DataRowSmall'
+import DataRowLarge from './DataRowLarge'
+import RowItem from './RowItem'
+import RowItemSpin from './RowItemSpin'
+import { CommonTableProps } from './types'
+import RowItemWrapperSmall from './RowItemWrapperSmall'
+import RowItemWrapperLarge from './RowItemWrapperLarge'
 
 size.defaultOptions({
   precision: 2,
 })
 
-export const CommonTable: FC<CommonTableProps<any>> = ({
+export const CommonTable: FC<
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  CommonTableProps<any>
```

Comment:
I'd like to figure out what the proper type here is, but did this for expedience.